packages/app-degree-pages/src/core/utils/http-url-resolver.js

```diff
@@ -17,16 +17,40 @@ function urlResolver(dataSource, defaultDataSource) {
     httpParameters["collegeOrg"] = httpParameters["collegeAcadOrg"];
     delete httpParameters["collegeAcadOrg"];
   }
+  if (httpParameters.program) {
+    // Convert `program` (from props) to `degreeType` for use as paramter
+    // to accommodate Data Potluck API changes.
+    const { program } = httpParameters;
+    httpParameters["degreeType"] = program === "undergrad" ? "UG" : "GR";
+    delete httpParameters["program"];
+  }
+  if (httpParameters.acadPlan) {
+    httpParameters.endpoint += `/${httpParameters.acadPlan}`;
+    delete httpParameters["acadPlan"];
+  }
+  if (httpParameters.cert === "true") {
+    httpParameters.degreeType = "UGCM";
+    delete httpParameters["cert"];
```

Comment:
Dropping a comment here to find my way back, in case any adjustments may be needed based on Undergrad and Graduate cert handling differences.